common/jsonquery/converters.go

```diff
@@ -66,3 +66,14 @@ func (convertor) ObjectToMap(node *ajson.Node) (map[string]any, error) {
 
 	return result, nil
 }
+
+// NodeToArray treats current node as an array of objects.
+// Sometimes JSON response from the server right of the bet is an array. This method performs the necessary conversion.
+func (convertor) NodeToArray(node *ajson.Node) ([]*ajson.Node, error) {
```

Comment:
yeah but we still don't need this wrapper. 

`arr, err := jsonquery.Convertor.NodeToArray(res.Body)` this can be
`arr, err := res.Body.GetArray()` then we don't need this layer. Also this loses original message. 

Only difference is that the error message changes. If we really need `ErrNotArray` we can just wrap.

Reply:
What are the benefits?